ui/raidboss/data/06-ew/raid/p12s.txt

```diff
@@ -4,3 +4,169 @@ hideall "--Reset--"
 hideall "--sync--"
 
 0.0 "--Reset--" sync / 21:........:4000000F:/ window 100000 jump 0
+
+# P1 Athena
+0.0 "--sync--" sync /Engage!/ window 0,1
+1.0 "--sync--" sync / 03:[^:]*:Hemitheos:00:5A:0{4}:00::12383:/ window 10,3 jump 1000.5 # Sync to P2 immediately through AddCombatant.
```

Comment:
Ooh this is a nice find.

Reply:
There's a similar AddCombatant for P1, but no need to jump ahead if already in P1 & the timing on them can be a little offset. No more than by a fraction of a second, but it is there, hence the additional sync in P2 when Pallas starts channeling the initial raidwide.